src/rest-server/src/middlewares/v2/protocol.js

```diff
@@ -16,6 +16,9 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 
+// skip guard-for-in becasue protocolValidate has already filtered keys
```

Comment:
Why not `for (key of Object.keys(obj))` ?

Reply:
Updated.